addon-test-support/@ember/test-helpers/index.ts

```diff
@@ -50,9 +50,12 @@ export { default as registerDebugInfoHelper } from './-internal/debug-info-helpe
 export type { TestMetadata } from './test-metadata';
 export { default as getTestMetadata } from './test-metadata';
 export {
-  registerHook as _registerHook,
-  runHooks as _runHooks,
-} from './-internal/helper-hooks';
+  registerHook,
+  runHooks,
+  type Hook,
+  type HookLabel,
+  type HookUnregister,
+} from './helper-hooks';
```

Comment:
@rwjblue I think a new release of both will resolve the issue, yes? Coordinating a major version bump for this package now, and then it can be updated in `ember-a11y-testing`.

Reply:
@MelSumner we absolutely do not want to do a major of this right now. We have other TS things in flight which we want to land backwards-compatibly before that. This doesn't need a breaking change to release, it just needs a deprecation-and-not-removal here.